.github/workflows/cd.yml

```diff
@@ -0,0 +1,51 @@
+name: Business Filing UI CD
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      environement:
+        description: "Environment (dev/test)"
```

Comment:
Dev only?

Reply:
no, it can be deploy to test use this github event.

Reply:
Ah, I see. Or any environment I guess?